### DISCUSSION TITLE
I can't get into my Ant Design pro account

### DISCUSSION THREAD

**Discussion by KirillNovikovvv** [ant-design/ant-design#51403] [General]
Hello!
I have previously worked on two projects with the free version of Ant. Some time ago I paid with the guys from the Ultimate Package startup. I received a notification by email confirming the payment, the letter contains a license key. But I have been struggling with how to use it for a day now. Can anyone please tell me where I should apply the key and log into my Ant Design account?

**Comment by afc163** [comment]
Ant Design Pro is **FREE** to use under MIT license, we never send paid license key in any way: https://github.com/ant-design/ant-design-pro/blob/master/LICENSE

**Comment by zombieJ** [comment]
Ant Design is open source project. It is free.
